debugger/src/cell_renderers/cellrendererbreakicon.c

```diff
@@ -279,12 +289,16 @@ static gint cell_renderer_break_icon_activate(GtkCellRenderer *cell, GdkEvent *e
 static void cell_renderer_break_icon_init (CellRendererBreakIcon *cell)
 {
 	GtkCellRenderer *cell_renderer = (GtkCellRenderer*)cell;
+	GValue mode = G_VALUE_INIT;
 	
 	cell->enabled = TRUE;
 	cell->condition = NULL;
 	cell->hitscount = 0;
-	
-	cell_renderer->mode = GTK_CELL_RENDERER_MODE_ACTIVATABLE;
+
+	g_value_init(&mode, G_TYPE_ENUM);
```

Comment:
This should be:

    g_value_init(&mode, GTK_TYPE_CELL_RENDERER_MODE);

Otherwise you get errors like this:

````
(geany:16953): GLib-GObject-WARNING **: 00:27:20.736: ../../../../gobject/gvalue.c:188: cannot initialize GValue with type 'GEnum', this type is abstract with regards to GValue use, use a more specific (derived) type

(geany:16953): GLib-GObject-CRITICAL **: 00:27:20.736: g_value_set_enum: assertion 'G_VALUE_HOLDS_ENUM (value)' failed

(geany:16953): GLib-GObject-CRITICAL **: 00:27:20.736: g_value_transform: assertion 'G_IS_VALUE (src_value)' failed

(geany:16953): GLib-GObject-WARNING **: 00:27:20.736: unable to set property 'mode' of type 'GtkCellRendererMode' from value of type '(null)'
````